src/components/dashboard/FaceVerification/screens/IntroScreen.js

```diff
@@ -184,27 +185,27 @@ const getStylesFromProps = ({ theme }) => ({
     width: '100%',
   },
   mainTitle: {
-    marginTop: getDesignRelativeHeight(isBrowser ? 30 : 15),
+    marginTop: getDesignRelativeHeight(isLargeDevice ? 30 : 10),
   },
   illustration: {
     marginTop: getDesignRelativeHeight(18),
     marginBottom: getDesignRelativeHeight(18),
-    height: getDesignRelativeWidth(isBrowser ? 220 : 130),
+    height: getDesignRelativeWidth(isLargeDevice ? 220 : 180),
     width: '100%',
   },
   descriptionContainer: {
     paddingHorizontal: getDesignRelativeHeight(theme.sizes.defaultHalf),
-    paddingVertical: getDesignRelativeHeight(isBrowser ? theme.sizes.defaultDouble : 14),
+    paddingVertical: getDesignRelativeHeight(isLargeDevice ? theme.sizes.defaultDouble : 14),
   },
   descriptionUnderline: {
-    display: 'block',
-    paddingTop: getDesignRelativeHeight(isBrowser ? theme.sizes.defaultQuadruple : theme.sizes.defaultDouble),
+    display: Platform.select({ web: 'block', default: 'flex' }),
+    paddingTop: getDesignRelativeHeight(isLargeDevice ? theme.sizes.defaultQuadruple : theme.sizes.defaultDouble),
   },
   button: {
     width: '100%',
   },
   bottomSeparator: {
-    marginBottom: getDesignRelativeHeight(25),
+    marginBottom: isLargeDevice ? getDesignRelativeHeight(25) : getDesignRelativeHeight(5),
```

Comment:
Why do we need this?
The margin to the 'ok, verify me' button should be 25 on every device. See how it looks after your changes (https://prnt.sc/ucybze).
We are using getDesignRelativeHeight to make the sizes responsive.

Reply:
mmm yeah it's not necessary